src/EditorFeatures/Test2/Rename/RenameEngineTests.vb

```diff
@@ -5926,6 +5926,24 @@ End Class
             End Using
         End Sub
 
+        <WorkItem(8297, "https://github.com/dotnet/roslyn/issues/8297")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Rename)>
+        Public Sub RenameVBClassNamedNew()
+            Using result = RenameEngineResult.Create(_outputHelper,
+                    <Workspace>
+                        <Project Language="Visual Basic" AssemblyName="Project1" CommonReferences="true">
+                            <Document><![CDATA[
+Class [|[$$New]|]
+    Sub New()
+    End Sub
+End Class
+]]>
+                            </Document>
+                        </Project>
+                    </Workspace>, renameTo:="New2")
+            End Using
```

Comment:
Oh duh.  Sorry, thought this was a case where we just disable rename.  I can see it renames, just not the constructor.  Can you have another reference to show that that other reference (i.e. not hte decl) does get renamed?